docs/projects/ad9265_fmc/index.rst

```diff
@@ -0,0 +1,230 @@
+-------------------------------------------------------------------------------
+
+The :adi:`AD9265` is a 16-bit, 125 MSPS analog-to-digital converter (ADC). The 
+:adi:`AD9265` is designed to support communications applications where high 
+performance combined with low cost, small size, and versatility is desired. The 
+ADC core features a multistage, differential pipelined architecture with 
+integrated output error correction logic to provide 16-bit accuracy at 125 MSPS 
+data rates and guarantees no missing codes over the full operating temperature 
+range. The ADC output data format is either parallel 1.8 V CMOS or LVDS (DDR). 
+A data output clock is provided to ensure proper latch timing with receiving 
+logic. The board also provides other options to drive the clock and analog 
+inputs of the ADC.
+
+Supported boards
+-------------------------------------------------------------------------------
+
+-  :adi:`EVAL-AD9265 <EVAL-AD9265>`
+
+Supported devices
+-------------------------------------------------------------------------------
+
+-  :adi:`AD9265`
+
+Supported carriers
+-------------------------------------------------------------------------------
+
+.. list-table::
+   :widths: 35 35 30
+   :header-rows: 1
+
+   * - Evaluation board
+     - Carrier
+     - FMC slot
+   * - :adi:`EVAL-AD9265-FMC-500EBZ <EVAL-AD9265>`
+     - :xilinx:`ZC706`
+     - FMC LPC
+   * -
+     - :xilinx:`ZedBoard <products/boards-and-kits/1-8dyf-11.html>`
+     - FMC LPC
+
+Block design
+-------------------------------------------------------------------------------
+
+Block diagram
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. image:: ad9265_fmc_block_diagram.svg
+   :width: 800
+   :align: center
+   :alt: AD9265/ZedBoard block diagram
+
+Clock scheme
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+There are 3 ways to configure the clock source for :adi:`AD9265` (some 
+modification maybe necessary).
+
+- External passive clock (default)
+- Optional active clock path using the :adi:`AD9517`
+- Optional oscillator
+
+For more details, check :adi:`AD9265` schematic.
+
+
+CPU/Memory interconnects addresses
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The addresses are dependent on the architecture of the FPGA, having an offset
+added to the base address from HDL (see more at :ref:`architecture`).
+
+Depending on the values of parameters $INTF_CFG, $ADI_PHY_SEL and $TDD_SUPPORT,
+some IPs are instatiated and some are not.
+
+Check-out the table below to find out the conditions.
+
+==================== =============== 
+Instance             Zynq/Microblaze
+==================== =============== 
+axi_ad9265           0x44A00000         
+axi_ad9265_dma       0x44A30000     
+==================== =============== 
+
+SPI connections
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+THESE ARE JUST EXAMPLES!!!
```

Comment:
I would rather:
```
.. note::

    The values below are just examples, use whichever fits best your case.
```

Reply:
These lines should be removed, as they are from the documentation template....